Sprint_04

Ka1der opened the pull request:
Сделал пару фиктивных изменений для корректного залива на GitHub  и создания Pull Request

Ka1der committed:
Сделал пару фиктивных изменений для корректого залива на GitHub  и создания Pull Request

helecta commented on MovieQuiz/Presentation/MovieQuizViewController.swift:
```diff
@@ -1,72 +1,139 @@
+//Outlets
+    @IBOutlet private var imageView: UIImageView!
+    @IBOutlet private var textLabel: UILabel!
+    @IBOutlet weak var counterLabel: UILabel!
```
Стоит организовывать код следующим образом:

- Публичные переменные
- IBOutlet
- Приватные переменные
- Публичные методы(сначала `init`ы, `override`ы, и потом остальные)
- Приватные методы
- IBAction

Подробнее почитать можно тут: [Организация кода](https://github.com/tutu-ru/swift-style-guide?tab=readme-ov-file#Организация-кода)

helecta commented on MovieQuiz/Presentation/MovieQuizViewController.swift:
```diff
@@ -1,72 +1,139 @@
+            DispatchQueue.main.asyncAfter(deadline: .now() + 1.0) {
+                self.showNextQuestionOrResults()
+            }
+        }
```
Не забывай следить за отступами, общепринятые отступы помогают читать код быстрее. В общем случае поможет выделить весь файл cmd+A и зажать control+i.

helecta commented on MovieQuiz/Presentation/MovieQuizViewController.swift:
```diff
@@ -1,72 +1,139 @@
+        imageView.layer.masksToBounds = true
+        imageView.layer.borderWidth = 10
+        imageView.layer.borderColor = UIColor.white.cgColor
+        imageView.layer.cornerRadius = 10
```
в макете 20

helecta commented on MovieQuiz/Presentation/MovieQuizViewController.swift:
```diff
@@ -1,72 +1,139 @@
+        imageView.layer.cornerRadius = 10
+        imageView.layer.borderColor = isCorrect ? UIColor.ypGreen.cgColor : UIColor.red.cgColor
+            DispatchQueue.main.asyncAfter(deadline: .now() + 1.0) {
+                self.imageView.layer.borderColor = UIColor.ypWhite.cgColor
```
белой рамки в макете нет, по заданию:

> После выбора ответа на вопрос через 1 секунду автоматически появляется следующий вопрос и рамка ответа исчезает

Ka1der committed:
- откорректировал шрифт счетчика вопросов
- откорректировал расположение текстового поля с вопросом
- сделал сгругления углов у постера 20px в interface builder
- сделал толщину и скругления углов рамки согласно макету
- прописал цвета из макета и применил их к рамке с правильным и не правильным варинатом ответов
- удалил белую рамку
- после выбора ответа через 1 секунду появляется новый вопрос и рамка исчезает
- откорректировал цвет иконок в статус баре на белый
- наименование кнопок "Да" и "Нет" исправил согласно макета (без капслока)
- сделал выключение и включение кнопок с помощью isEnabled

Ka1der committed:
- откорректировал расположение текстового поля с вопросом
- сделал сгругления углов у постера 20px в interface builder
- сделал толщину и скругления углов рамки согласно макету
- прописал цвета из макета и применил их к рамке с правильным и не правильным варинатом ответов
- удалил белую рамку
- после выбора ответа через 1 секунду появляется новый вопрос и рамка исчезает
- откорректировал цвет иконок в статус баре на белый
- наименование кнопок "Да" и "Нет" исправил согласно макета (без капслока)
- сделал выключение и включение кнопок с помощью isEnabled

Ka1der commented:
Привет дорогой ревьюер! Спасибо за развернутые комментарии к замечаниям. Выше будет два практически одинаковых коммита потому что я пушу на гитхаб через Xcode и в первый раз случайно запушил не откорректированную версию. Я новичок и с гитхабом не успел до конца разобраться. Сейчас в ветке sprint_04 актуальная версия с откорректированным проектом по твоим замечаниям. Жду результатов ревью. Заранее спасибо)
